SpatialGDK/Documentation/content/get-started/build-unreal-fork.md

```diff
@@ -80,4 +82,5 @@ Use as a base for creating your own project running on SpatialOS.
 <br/>
 
 ------</br>
-_2019-05-30 Page updated with editorial review_
+_2019-08-03 Page updated with editorial review: added clarification on SSH key and Linux dependencies._
```

Comment:
Added this direct to the branch - hope this is OK.

Reply:
I know there have been other edits since May as I can see the sticking plasters in the text. :-)